client/components/account-balances/test/index.test.tsx

```diff
@@ -146,7 +147,7 @@ const createMockOverview = (
 			date: Date.now(),
 			fee: 0,
 			fee_percentage: 0,
-			status: 'scheduled',
```

Comment:
This 'scheduled' status was in a mock deposit. It's not a legitimate status so I've updated it to `'estimated'`. It didn't have any bearing on the tests itself.